.vib/nginx/goss/goss.yaml

```diff
@@ -0,0 +1,40 @@
+file:
+  /opt/bitnami/nginx/conf/nginx.conf:
+    exists: true
+    filetype: file
+    mode: "0664"
+    owner: root
+    group: root
+    contains:
+      - /listen\s+8080/
+  /opt/bitnami/nginx/html/index.html:
+    exists: true
+    filetype: file
+    mode: "0644"
+    owner: root
+    group: root  
+command:
+  user-id-test:
+    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+    exit-status: 0
+    stdout: []
+    stderr: [] 
+  nginx-config-files:
+    exec: nginx -t 
+    exit-status: 0
+    stdout: []
+    stderr: 
+      - "nginx.conf syntax is ok"
+      - "nginx.conf test is successful"  
+  nginx-reload:
+    exec: nginx -s reload
+    exit-status: 0
+    stdout: []
+    stderr: 
+      - "signal process started"
+  nginx-curl:
+    exec: curl localhost:8080
```

Comment:
If we are checking the `stdout` for a specific text, I'd change the command to something more dynamic:

```
diff <(curl --silent localhost:8080) /opt/bitnami/nginx/html/index.html
```

This will check that the index served by the server is actually the same as the one on disk (and it is fail-proof in case the text changes in the future!)

WDYT?

Reply:
I like the idea but this command returns status code 2. 
I'll try to debug it.
```

        "test_id" : "goss-0-11",
        "name" : "Command nginx-curl exit-status",
        "status" : "FAILED",
        "message" : "ERRORS: [Command: nginx-curl: exit-status: doesn't match, expect: [0] found: [2]] FAILURES: [Command: nginx-curl: exit-status: doesn't match, expect: [0] found: [2]] ",
```

Reply:
Removing this test as agreed due to the fact we shouldn't be using `curl`.